pkg/maven/settings.go

```diff
@@ -0,0 +1,149 @@
+package maven
```

Comment:
There is another PR for settings file handling (PR #1176). These files are supposed to disappear from this PR when the other PR gets merged (that PR will be rebased on top of the other).